src/submission/submission-error-messages.ts

```diff
@@ -45,7 +45,10 @@ const ERROR_MESSAGES: { [key: string]: (errorData: any) => string } = {
     `${errorData.info.value} has not yet been registered. Please register samples before submitting clinical data for this identifier.`,
   CONFLICTING_TIME_INTERVAL: () =>
     'survival_time cannot be less than Specimen specimen_acquisition_interval.',
-
+  FOLLOW_UP_CONFLICING_INTERVAL: () =>
+    'interval_of_followup cannot be less than Treatment treatment_start_interval.',
+  TREATMENT_TIME_CONFLICT: () =>
+    'Treatment treatment_start_interval cannot be greater than FollowUp interval_of_followup.',
```

Comment:
you  can use the same error message and code I believe FOLLOWUP_TREATMENT_ INTERVAL_CONFLICT